app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt

```diff
@@ -153,6 +158,19 @@ fun ActivityDetailsScreen(
     listActivityViewModel.updateActivity(activity!!.copy(comments = comments))
   }
 
+  val creatorID = activity?.creator ?: ""
+  var creator by remember {
+    mutableStateOf(User(creatorID, "anonymous", "anonymous", listOf(), listOf(), "", listOf()))
+  }
+  activity?.participants?.forEach() {
+    if (it.id == creatorID) {
+      creator = it
+    }
+  }
+  val uiState by listActivityViewModel.uiState.collectAsState()
+  val activitiesList = (uiState as ListActivitiesViewModel.ActivitiesUiState.Success).activities
+  val nbActivitiesCreated = activitiesList.filter { it.creator == creator.id }.size
```

Comment:
This is a smart way to do it. However it iterates on all the activities, which could take a lot of time. As we don't have much time to develop features in the app, we can keep this as it is. But a further developement can be making nbActivitiesCreated an attribute of the profile